src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -158,17 +155,14 @@ for live streaming and video on demand for the web.
 You can use the same method to create a key for both DASH and HLS. Do this using
 openssl. The following will create a key made of 16 hex values.
  
-    openssl rand -out media.key 16
+    openssl rand -hex -out media.key 16
     
-This command creates a file with white space and new line characters, which are
-not allowed by Shaka Packager. You'll need to open the key file and manually
-remove all whitespace including the final carriage return.
-
 ### Encrypt
 
-For the `-key` flag use the key created earlier and stored in the media.key
-file. However, when entering it on the command line, be sure you've removed its
-whitespace. For the `-key_id` flag repeat the key value.
+The `-key` flag is copied from the `media.key` file. The `-key_id` flag is
```

Comment:
I actually explained here why key_id and key are 16 bytes: https://github.com/google/WebFundamentals/pull/4682/files/39197b52d046b47737575271d0f9ef4302476391#r122334500.

This paragraph should be rephrased. Maybe something like:

Both key and key-id are required to be 16 bytes, i.e. 32 hex digits.